dmutils/logging.py

```diff
@@ -148,6 +150,8 @@ def process_log_record(self, log_record):
             ("span_id", "spanId",),
             ("parent_span_id", "parentSpanId",),
             ("app_name", "application",),
+            ("tracing_is_sampled", "tracingIsSampled",),
+            ("tracing_debug_flag", "tracingDebugFlag",),
```

Comment:
I think there's a test for these field renamings that we can add these too?

Reply:
There are tests but are we going to add a test for every single pair we add here?

Reply:
Oh I think I see the bit you mean...

Reply:
I'm hoping we're not going to add too many more!